src/term.rs

```diff
@@ -513,23 +527,20 @@ fn show_precedence_cla(
     match term {
         Var(0) => "undefined".to_owned(),
         Var(i) => {
-            if depth >= *i as u32 {
-                from_u32(depth + 97 - *i as u32)
-                    .expect("error while printing term")
-                    .to_string()
+            let cast = *i as u32;
```

Comment:
```suggestion
            let i = *i as u32;
```
I'd prefer to shadow here

Reply:
This is done